FluentValidation-Draft.md

```diff
@@ -166,9 +166,134 @@ You can also throw exceptions, by using ValidateAndThrow, like this:
 ```c#
 using FluentValidation;
 ...
-validator.ValidateAndThrow(googleUserInfo)`
+validator.ValidateAndThrow(googleUserInfo)
+```  
+
+
+
+### Configuration checker
+
+ With fluent validation we can validate **any** class, and there is nothing stopping us from using it to validate 
+configuration properties in the project. This is a very neat way of checking if all the configuration is set up before we fire up the solution. 
+The project will build normally, but when started it will throw an error and remind us of which configuration we are missing.
+It's a really good solution for the dev-ops team when they are setting application configuration.
+We will also bypass annoying unspecified errors which we usually get when the configuration is missing. 
+
+**How to do it?**
+
+
+- Take all of the classes which are used in the IOptions pattern (this is an example)
+```c#
```

Comment:
If you don't add an empty line between the text and a code block, usually that results in a messed up formatting.

Reply:
Handbook police 🚓 👮‍♂️ 😄

Reply:
🚨